main.go

```diff
@@ -119,14 +120,23 @@ func main() {
 	}
 }
 
-func serve(ctx context.Context, db *DB, keyset *derivedKeyset, issuer issuerConfig, _, _ time.Duration, addr string) error {
+func serve(ctx context.Context, db *DB, appKeys *derivedKeyset, issuer issuerConfig, _, _ time.Duration, addr string) error {
```

Comment:
Can the two unused time.Duration params be deleted now?

Reply:
removed